src/hm2.ts

```diff
@@ -0,0 +1,165 @@
+interface Lecturer {
+  name: string;
+  surname: string;
+  position: string;
+  company: string;
+  experience: number;
+  courses: string;
+  contacts: string;
+}
+
+class School {
+  _areas: string[] = [];
+  _lecturers: string[] = [];
+
+
+  get areas(): string[] {
+    return this._areas;
+  }
+
+  get lecturers(): string[] {
+    return this._lecturers;
+  }
+
+
+  addArea(area: string): void {
+    this._areas.push(area);
+  }
+
+  removeArea(area: string): void {
+    const index = this._areas.indexOf(area);
+    if (index !== -1) {
+      this._areas.splice(index, 1);
+    }
+  }
+
+  addLecturer(lecturer: string): void {
+    this._lecturers.push(lecturer);
+  }
+
+  removeLecturer(lecturer: string): void {
+    const index = this._lecturers.indexOf(lecturer);
+    if (index !== -1) {
+      this._lecturers.splice(index, 1);
+    }
+  }
+}
+
+
+class Area {
+  _levels: any[] = [];
```

Comment:
You already know how to add type annotations for objects. The use of `any` type is not acceptable.